src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -291,7 +291,7 @@ public string GetNodeStats()
                     "Peer:" + (node.RemoteInfo() + ", ").PadRight(LoggingConfiguration.ColumnLength + 15) +
                     (" connected" + " (" + (connectionManagerBehavior.Inbound ? "inbound" : "outbound") + "),").PadRight(LoggingConfiguration.ColumnLength + 7) +
                     (" agent " + agent + ", ").PadRight(LoggingConfiguration.ColumnLength + 2) +
-                    " height=" + chainHeadersBehavior.PendingTip.Height);
+                    " height=" + chainHeadersBehavior.GetPendingTipOrChainTip().Height);
```

Comment:
again why?

Reply:
Because here GetPendingTipOrChainTip() can return our tip if the pending one is null. And PendingTip now can be null if the peer didn't send us headers or we didn't send them to the peer.

Reply:
Now how this is not misleading - if you said in above comment that we make a false image of peer's tip, this is exactly where you revert that logic again to that misleading value. It does not make sense to me. Either you want to really redefine the pending tip to hold the actual value, if we have it, or null, if we don't, then you should display here "unknown" if it is null and not this.

Reply:
The thing is that after we're synced and connected to several nodes we get half of them at the "unknown" state because no headers were transferred between. So here its actually logical go assume that we and peer are at the same height.

Reply:
i think having unknown is much better than assumed information that may or may not be anywhere close to the real thing

Reply:
Ok, reasonable. I'll turn it back to "unknown" if the pending tip is null.